core/session.rb

```diff
@@ -267,11 +269,35 @@ def showBoxes
     return exit_code
   end
 
+  def showBoxField
```

Comment:
Please refactor this functions into two functions:
-- findBoxField(field, boxName) - this function should generate exception in case of "Box #{$session.boxName} is not found" or "Box #{$session.boxName} does not have #{$session.field} key"
-- showBoxField which wraps findBoxField and return 1 in case of exception